projects/plugins/jetpack/tools/webpack.config.extensions.js

```diff
@@ -268,4 +301,53 @@ module.exports = [
 			} ),
 		],
 	},
+	{
```

Comment:
Build assets (JS and CSS) for "single" blocks. It also copies the `block.json` file.